packages/core/jest.config.js

```diff
@@ -2,4 +2,8 @@ module.exports = {
   ...require("../../jest.config"),
   // Allows running tests from packages/core/
   testEnvironment: "<rootDir>/../../tests/environment/customEnvironment.js",
+  moduleNameMapper: {
+    // Here, the OIDC package is used, but any package depending on jose@3.x would do.
+    "^jose/(.*)$": "<rootDir>/node_modules/jose/dist/node/cjs/$1",
+  },
```

Comment:
This shouldn't be necessary in the `-core` package, because we should not be using plain `jose` there - even before #1516, as this package is a dependency of `-browser`.

Reply:
30eba5f9348fc6eecbf3e124e83eea4f26397248 It wasn't actually depending on `jose`, but only due to the module mapping that tricked jest into having the appropriate behaviour ^^